ruler/ruler.go

```diff
@@ -17,218 +15,165 @@ import (
 	"github.com/weaveworks/cortex/distributor"
 	"github.com/weaveworks/cortex/querier"
 	"github.com/weaveworks/cortex/user"
-	"github.com/weaveworks/cortex/util"
 )
 
+var (
+	evalDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "cortex",
+			Name:      "group_evaluation_duration_seconds",
+			Help:      "The duration for a rule group to execute.",
+		},
+	)
+	rulesProcessed = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "rules_processed",
```

Comment:
suffix _total according to best practices? https://prometheus.io/docs/practices/naming/